CoinDock-app/src/Screens/Account/Account.js

```diff
@@ -8,37 +8,90 @@ import { useLogout } from "App/Api/auth";
 import "Shared/common-styles/common.css";
 
 function Account() {
-  const { data: account, isLoading } = useAccount();
-  const [logout] = useLogout();
-  const navigate = useNavigate();
+ const { data: account}= useAccount();
 
-  const handleLogoutClick = async () => {
-    try {
-      await logout().unwrap();
-      navigate("/login");
-    } catch (e) {
-      navigate("/login");
-    }
-  };
+ const accountDetails = account?.result?.user || {};
+ const [logout] = useLogout();
+ const navigate = useNavigate();
 
-  return (
+ const accordianBasedAccountDetails = [
+   {
+     label:'Profile Settings',
+     field:[
+       {
+       label:'Name',
+       fieldKey:'name',
+       type:'edit'
+      },
+       {
+        label:'DateofBirth',
+        fieldKey:'dateofbirth',
+        type:'edit'
+       },
+       {
+        label:'Country',
+        fieldKey:'country',
+        type:'edit'
+       },
+    ]
+   },
+   {
+     label:'Account Settings',
+     field:[
+         {
+          label:'Email',
+          fieldKey:'email',
+          type:'edit'
+         },
+         {
+          label:'Change Password',
+          fieldKey:'changePassword',
+          type:'edit'
+         },
+         {
+          label:'Recovery Code',
+          fieldKey:'recoverycode',
+          navigate:true,
+         },
+
+    ]
+   },
+     {
+     label:'System Settings',
+     field:[
+       {
+         label:'Primary currency',
+         fieldKey:'primarycurrency',
+       },
+       {
+        label:'Secondry currency',
+        fieldKey:'secondrycurrency',
+      }
+     ]
+     }
+  ]
+
+ const handleLogoutClick = async () => {
+  try {
+    await logout().unwrap();
+    navigate("/login");
+  } catch (e) {
+    navigate("/")
+  }
+};
+return (
     <div className="container-1 justify-content-center align-items-center">
-      <div className="col-md-4 col py-5 ">
-        {isLoading ? "...LOADING" : ""}
-        {account &&
-          account.map((accs, id) => {
-            return <Accordion key={id} name={accs.name} items={accs.items} />;
-          })}
-      </div>
-      <div className="d-flex justify-content-start">
-        <button
-          className="cd-button cd-button-2 cd-logout-button"
-          onClick={handleLogoutClick}
-        >
-          Logout
-        </button>
-      </div>
-    </div>
-  );
-}
+         <div className="col-md-4 col py-5 ">
+         {accordianBasedAccountDetails && accordianBasedAccountDetails.map((item,id) => (
+               <div>
+                  <Accordion key={id} label={item.label} field={item.field} value={accountDetails} />
+                </div>
+          ))}
+       </div><div className='d-flex justify-content-start'>
```

Comment:
can you please format the code over here please.
I suggest to use prettier or other formatters extension for formatting. So code can be readable.